### PR TITLE
Bug fixes to facilitate uluru migration

### PR DESCRIPTION
**Pull request by nbharad**
*Issue #, if available:*
Bug fixes to facilitate uluru migration

*Description of changes:*
Some bug fixes based on recommendation from CFN team

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: aws-ssm-parameter/src/main/java/com/amazonaws/ssm/parameter/BaseHandlerStd.java [hunk 1]
```diff
@@ -88,6 +94,11 @@ protected static boolean stabilize(
         if(response == null || response.invalidParameters().size() != 0) {
             return false;
         }
+
+        // Temporary logging, TODO: Remove these logs after debugging
```
Comment (joshuadeanhall) [inline aws-ssm-parameter/src/main/java/com/amazonaws/ssm/parameter/BaseHandlerStd.java:98]: How long do you plan to keep this logging?

Comment (nbharad) [reply aws-ssm-parameter/src/main/java/com/amazonaws/ssm/parameter/BaseHandlerStd.java:98]: Probably till the migration is completed, once it is done, we plan on removing this

**Review by baoqianz123 — APPROVED** [review]

**Review by joshuadeanhall — APPROVED** [review]

**Review by RockyYan1994 — APPROVED** [review]
LGTM

File: aws-ssm-parameter/src/main/java/com/amazonaws/ssm/parameter/BaseHandlerStd.java (hunk 1 above)
Comment (sahithj) [reply aws-ssm-parameter/src/main/java/com/amazonaws/ssm/parameter/BaseHandlerStd.java:98]: Why are we logging GetParameter response?

Comment (nbharad) [reply aws-ssm-parameter/src/main/java/com/amazonaws/ssm/parameter/BaseHandlerStd.java]: Updated to only log name and version from get parameters response

**Review by sahithj — APPROVED** [review]

**Pull request merged**